.github/workflows/continuous-integration-workflow.yml

```diff
@@ -308,45 +308,10 @@ jobs:
     steps:
       - name: Checkout the repo
         uses: actions/checkout@v2
-      - name: Configure AWS credentials
-        uses: aws-actions/configure-aws-credentials@v1
-        with:
-          aws-access-key-id: '${{ secrets.GHA_AWS_ACCESS_KEY_ID }}'
-          aws-secret-access-key: '${{ secrets.GHA_AWS_SECRET_ACCESS_KEY }}'
-          aws-region: '${{ secrets.AWS_DEFAULT_REGION }}'
-          role-to-assume: '${{ secrets.GHA_AWS_ROLE_TO_ASSUME }}'
-          role-duration-seconds: 1200
-          role-session-name: 'temp-session'
-          role-skip-session-tagging: true
-      - name: Docker login to ECR
-        uses: aws-actions/amazon-ecr-login@v1
-        id: login-ecr
-      # https://github.com/docker/build-push-action/issues/20
-      - name: Get ecr password
-        id: get-ecr-password
-        run: |
-          aws ecr get-login-password \
-          | {
-            read PASSWORD
-            echo "::add-mask::$PASSWORD"
-            echo "::set-output name=password::$PASSWORD"
-            }
-      - name: Login to DockerHub
-        uses: docker/login-action@v1
-        with:
-          username: ${{ secrets.DOCKER_READONLY_USERNAME }}
-          password: ${{ secrets.DOCKER_READONLY_PASSWORD }}
```

Comment:
This step was there to prevent Docker Hub from rate-limiting base image pulls AFAIK, so if you start having issues with this workflow not being able to pull from Docker Hub, try adding it back.